reactor-netty-core/src/main/java/reactor/netty/udp/UdpClientConfig.java

```diff
@@ -126,12 +126,16 @@ protected ChannelMetricsRecorder defaultMetricsRecorder() {
 			AdvancedByteBufFormat.HEX_DUMP
 					.toLoggingHandler(UdpClient.class.getName(), LogLevel.DEBUG, Charset.defaultCharset());
 
-	static final class MicrometerUdpClientMetricsRecorder extends MicrometerChannelMetricsRecorder {
+	public static class MicrometerUdpClientMetricsRecorder extends MicrometerChannelMetricsRecorder {
 
 		static final MicrometerUdpClientMetricsRecorder INSTANCE = new MicrometerUdpClientMetricsRecorder();
 
-		MicrometerUdpClientMetricsRecorder() {
-			super(reactor.netty.Metrics.UDP_CLIENT_PREFIX, "udp");
+		public MicrometerUdpClientMetricsRecorder(String prefix) {
+			super(prefix, "udp");
+		}
+
+		public MicrometerUdpClientMetricsRecorder() {
+			this(reactor.netty.Metrics.UDP_CLIENT_PREFIX);
 		}
```

Comment:
Is there a value of making this public? One can directly use `MicrometerChannelMetricsRecorder`.

Reply:
ditto